src/services/webSocket.sevice.ts

```diff
@@ -0,0 +1,81 @@
+import * as WebSocket from 'ws';
+import { Injectable, Logger } from '@nestjs/common';
+import { ESocketError } from 'src/constant/exception.constant';
```

Comment:
Please fix the issues reported  by the Review BOT.

Reply:
Please review my commit. The test requires a node to run, so it will not work with the review BOT.